proposals/lip-0067.md

```diff
@@ -199,7 +199,9 @@ There are no incompatibilities since the protocol is not changed.
 
 ## Reference Implementation
 
-TBD
+- [Add semgrep config and fix identified issues #8593](https://github.com/LiskHQ/lisk-sdk/pull/8593)
```

Comment:
I think you should also remove the first two links, I don't see how they are connected with the LIP.